site-cookbooks/omnibus_sensu/recipes/default.rb

```diff
@@ -4,23 +4,64 @@
 #
 # Copyright (c) 2016 Sensu, All Rights Reserved.
 
-include_recipe "omnibus::default"
+include_recipe 'chef-sugar'
 
-case node["platform_family"]
-when "freebsd"
+if windows?
+  include_recipe 'chocolatey'
+
+  chocolatey 'dotnet3.5' do
+    version '3.5.20160716'
+  end
+
+  chocolatey 'windows-sdk-8.1' do
+    version '8.100.26654.0'
+  end
+
+  chocolatey 'microsoft-build-tools' do
+    version '14.0.25420.1'
+  end
+
+  # If this is an ephemeral vagrant/test-kitchen instance, we relax the password
+  # so that the default password "vagrant" can be used.
+  powershell_script 'Disable password complexity requirements' do
+    code <<-EOH
+      secedit /export /cfg $env:temp/export.cfg
+      ((get-content $env:temp/export.cfg) -replace ('PasswordComplexity = 1', 'PasswordComplexity = 0')) | Out-File $env:temp/export.cfg
+      secedit /configure /db $env:windir/security/new.sdb /cfg $env:temp/export.cfg /areas SECURITYPOLICY
+    EOH
+  end
+end
+
+if freebsd?
   package "git"
 else
   include_recipe "git"
+
+  git_config "user.email" do
+    value "support@sensuapp.com"
+    options "--global"
+  end
+
+  git_config "user.name" do
+    value "Sensu Omnibus Builder"
+    options "--global"
+  end
```

Comment:
Can we move these git_config lines out of the conditional statement? `user.email` and `user.name` will need to be set on all platforms.